README.md

```diff
@@ -23,9 +23,9 @@ The jsep built files will be in the build/ directory.
 
 #### Client-side
 
-```javascript
-<script src="/PATH/TO/jsep.min.js"></script>
-...
+```html
+<script src="/PATH/TO/jsep.iife.min.js"></script>
```

Comment:
I do think we should mention ESM too. They're not only useful for Node. Not sure if they should be before or after IIFE, I'll leave it up to you. I could argue this both ways: ESM should be first because it's the "canonical" version, or IIFE should be first because ESM might scare novices away.
Also, if we're including `<script>` elements, we should probably include one around the sample code.